rt_gene/src/rt_gene/extract_landmarks_method.py

```diff
@@ -117,6 +178,9 @@ def __init__(self, img_proc=None):
         self.facial_landmark_nn = face_alignment.FaceAlignment(landmarks_type=face_alignment.LandmarksType._2D,
                                                                device=self.device_id_facealignment, flip_input=False)
 
+        self.face_encoder = dlib.face_recognition_model_v1(
+            rospkg.RosPack().get_path('rt_gene') + '/model_nets/dlib_face_recognition_resnet_model_v1.dat') if face_encoding_tracker else None
```

Comment:
Does the dlib license allow to redistribute their model files? And would it be nicer to download rather than store it here in a similar way to https://github.com/Tobias-Fischer/rt_gene/blob/master/rt_gene/CMakeLists.txt#L24-L31? Anyways, we should add `dlib` to the list of libraries.

Reply:
Also, shouldn't this be a (static) attribute of the FaceEncodingTracker class? Then you also don't need to pass it around.

Reply:
It's licensed under the Boost license (similar to MIT), so all good license wise.

Reply:
> Does the dlib license allow to redistribute their model files? And would it be nicer to download rather than store it here in a similar way to https://github.com/Tobias-Fischer/rt_gene/blob/master/rt_gene/CMakeLists.txt#L24-L31? Anyways, we should add `dlib` to the list of libraries.

Agreed - added catkin_download but I figured we should upload it to the same location as RT-GENE's files? The files available to download from dlib itself are zipped and would require unzipping as a catkin command. I don't mind adding that command but figured it would maintain stability/ease of deployment to keep them together.